installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -6,35 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
-
-ifndef::openshift-origin[]
-
-[id="installing-sno-assisted-installer"]
-== Installing {sno} using the Assisted Installer
-
-To install {product-title} on a single node, use the web-based Assisted Installer wizard to guide you through the process and manage the installation.
-
-include::modules/install-sno-generating-the-discovery-iso-with-the-assisted-installer.adoc[leveloffset=+2]
-
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#persistent-storage-using-lvms_logical-volume-manager-storage[Persistent storage using logical volume manager storage]
-* xref:../../virt/about_virt/about-virt.adoc#virt-what-you-can-do-with-virt_about-virt[What you can do with OpenShift Virtualization]
-
-include::modules/install-sno-installing-with-the-assisted-installer.adoc[leveloffset=+2]
-
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../installing/installing_sno/install-sno-installing-sno.adoc#installing-with-usb-media_install-sno-installing-sno-with-the-assisted-installer[Creating a bootable ISO image on a USB drive]
-
-* xref:../../installing/installing_sno/install-sno-installing-sno.adoc#install-booting-from-an-iso-over-http-redfish_install-sno-installing-sno-with-the-assisted-installer[Booting from an HTTP-hosted ISO image using the Redfish API]
-
-* xref:../../nodes/nodes/nodes-sno-worker-nodes.adoc#nodes-sno-worker-nodes[Adding worker nodes to {sno} clusters]
-
-endif::openshift-origin[]
+See link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2023/html/assisted_installer_for_openshift_container_platform/index[Assisted Installer for OpenShift Container Platform] to install OpenShift Container Platform on a single node.
```

Comment:
Would it make more sense to  xref to this: https://docs.openshift.com/container-platform/4.14/installing/installing_sno/install-sno-preparing-to-install-sno.html since it seems like that is the content you want users to eventually navigate to.